samples/CookieSharing/README.md

```diff
@@ -0,0 +1,11 @@
+# Cookie Sharing Sample App
+
+The sample illustrates cookie sharing across three apps that use cookie authentication:
+
+| Project                                 | Description |
+| --------------------------------------- | ----------- |
+| CookieSharing.Core                      | ASP.NET Core 2.0 Razor Pages app without using ASP.NET Core Identity |
+| CookieSharing.WithIdentity.Core         | ASP.NET Core 2.0 MVC app with ASP.NET Core Identity |
+| CookieSharing.WithIdentity.NETFramework | ASP.NET Framework 4.6.1 MVC app with ASP.NET Identity |
+
```

Comment:
Maybe add a brief set of steps to demonstrate things are working

- Run the WithIdentity app, register a user, verify you are logged in
- Run the Core app, demonstrate that you are still logged in
- Run the old .NetIdentity app, demonstrate that you are still logged in

Reply:
I'm assuming everything has already been setup/wired up so they'd just have to run and go to the right url with the same brower session

Reply:
I'm not sure if we already have a topic for identity database interop as well, but your two samples here are perfect to demonstrate how you can use the same identity database between core + .NETFramework here as well.

Reply:
Correct